test/groovy/com/sap/piper/integration/TransportManagementServiceTest.groovy

```diff
@@ -129,15 +129,16 @@ class TransportManagementServiceTest extends BasePiperTest {
         def file = 'myFile.mtar'
         def namedUser = 'myUser'
 
-        shellRule.setReturnValue(JenkinsShellCallRule.Type.REGEX, ".* curl .*", '400')
+        // 418 (tea-pot)? Other than 400 which is used in verbose mode
```

Comment:
[minor] I would explain the reason for the different error code in the comment. And may be better would use another error response body, rather than another error code. Since this is our target - to expose the error body despite of the verbose flag and the test should check for that.

Reply:
ups, that is somehow incomplete. I guess I didn't save before committing. But currently we don't expose the error message in case we are non-verbose. That will change with your PR I guess.

Reply:
... done. The '418' is reflected in the log message we have also in the `verbose` mode. So we can clearly distinguish the log message in the verbose mode from the log message in the non verbose mode. In non-verbose mode there is no response body printed. Hence we can't use that.

Reply:
No, the error message is currently exposed in non-verbose mode in uploadFile(...) method. This test case even checks for that. That's why I proposed to provide different error messages for verbose and non-verbose cases. With my PR comes exposing of error messages in authentication(...) and uploadFileToNode(...) methods (and true, in both verbose and non-verbose cases).

Reply:
you are right. Fix now we provide different responses in both cases and check for that response in the test.